lib/timestamps_test.go

```diff
@@ -97,6 +97,91 @@ func TestGetTimestampsMulPages(t *testing.T) {
 	}
 	if !equals {
 		t.Fatalf("The expected timestamps %v and the actual ones %v"+
-			" don't have the sme values\n", expectedTimestamps, timestamps)
+			" don't have the same values\n", expectedTimestamps, timestamps)
+	}
+}
+
+func TestGetTimestampsDistributed(t *testing.T) {
+	filePathFormat := "testdata/distributed_stars_%d.json"
+	expectedTimestamps := 16
+	maxPage := 5
```

Comment:
You only have 4 files `lib/testdata/distributed_stars_*.json` so the maxPage here should actually be 4.
It is important when `GOMAXPROCS > 4` because then it falls back to `GetTimestamps` and uses the header, expects a 5th one but there isn't so the test fail.